### PR TITLE
Added converted files

### PR DESCRIPTION
**Pull request by mrshirts**
Will probably need some cleanup, but here's a draft of the rest of the converted files added. Had to do a lot of playing around with ParmEd first.

A couple of things to note:

* I've moved the previously converted GROMACS files, as they were lower precision and causing some issues in energy comparisons.  They have not been deleted, but moved to GROMACS_old.

* I haven't edited the InterMol output logfiles (with the energy comparisons) that much.

* Let me know if the description is enough.

### CODE REVIEW

**Comment by mrshirts** [comment]
This is the update version of PR #12.

File: SAMPLing_instructions.md [hunk 1]
```diff
@@ -95,5 +95,6 @@ All the host-guest system files in the `SAMPLing/` directory were prepared using
...
-- After the equilibration, the `System` was serialized into the OpenMM `xml` format. The `rst7` file was generated during the equilibration using the `RestartReporter` in the `parmed.openmm` module. The AMBER `prmtop` and `rst7` files were then converted to GROMACS `top`/`gro` and PDB formats by ParmEd and MDTraj `1.9.1` respectively.
+- The system was minimized with the L-BFGS optimization algorithm and equilibrated by running 1 ns of Langevin dynamics (BAOAB splitting, 1 fs time step) at 298.15K with a Monte Carlo barostat set at 1 atm using `OpenMM 7.1.1`. PME was used for long-range electrostatic interactions with a cutoff of 10A. VdW interactions used the same 10A cutoff and a switching distance of 9A.
+- After the equilibration, the `System` was serialized into the OpenMM `xml` format. The `rst7` file was generated during the equilibration using the `RestartReporter` in the `parmed.openmm` module (ParmEd `2.7.3`). The AMBER `prmtop` and `rst7` files were then converted to PDB format by MDTraj `1.9.1`.
+- Other coversions are described in `SAMPLing/conversion.md`, as well as validations of the energies calculated at each configuration between programs.
```
Comment (davidlmobley) [inline SAMPLing_instructions.md]: "coversions" -> "conversions"

Comment (davidlmobley) [reply SAMPLing_instructions.md]: Also, "validations of the energies calculated at each configuration between programs" is confusing; do you mean "after each CONVERSION between programs"? If not, what are the configurations in question?

File: host_guest/SAMPLing/conversion.md [hunk 2]
```diff
@@ -72,41 +80,44 @@ Programs used were:
...
+For the energy comparisons in `SAMPL6_energyoutput_conversion_longcutoff.txt`, the same command was used except for the following differences in the options: `--inefile runfiles/min_SAMPL6.in -gs runfiles/grompp_SAMPL6.mdp -ds runfiles/onepoint_SAMPL6.cfg -as runfiles/min_SAMPL6.in -ls pair_style lj/cut/coul/long 14.0 14.0\npair_modify tail yes\nkspace_style pppm 1e-8\n\n -cs nbond inbfrq -1 imgfrq -1 -\nelec ewald pmew fftx 48 ffty 48 fftz 48 kappa 0.22310095 order 4 -\nvdw vips cutnb 14. cutim 14. ctofnb 14. ctonnb 14.`
 
-The OPENMM, GROAMCS_old, and PDB files were converted using the
-default options of ParmEd (Git hash ???). The parameters for GROMACS are
+The OPENMM, GROMACS_old, and PDB files were converted using the
+default options of ParmEd (version or Git hash ???). The parameters for GROMACS are
```
Comment (davidlmobley) [inline host_guest/SAMPLing/conversion.md]: There's question marks here, need filling in?

File: host_guest/SAMPLing/conversion.md [hunk 3]
```diff
@@ -72,41 +80,44 @@ Programs used were:
...
+the difference in Coulomb 1-4 interactions, and approximately 70% of
+the difference in the total Coulomb interactions for standard cutoff
+schemes. This can be seen in the fact that the GROMACS, LAMMPS and
+DESMOND energies are generally always within 1 kJ/mol from each other,
+whereas the AMBER and CHARMM energies are further off (2-4 kJ/mol from
+the others.)
+
+We have not corrected the topology files for this difference in
+Coulomb's law constant, as it is not expected that these differences
+with significantly affect free energies calculated. The difference is
```
Comment (davidlmobley) [inline host_guest/SAMPLing/conversion.md]: "will significantly", not "with significantly"

**Comment by andrrizzi** [comment]
Looks good to me!

**Pull request merged**